test/puppetlabs/puppetdb/jdbc_test.clj

```diff
@@ -57,7 +57,8 @@
         (let [pool (-> *db*
                        defaulted-read-db-config
                        subject/pooled-datasource)]
-          (is (= 25 (.getMaximumPoolSize (:datasource pool))))))))
```

Comment:
@senior I collapsed the `let`s and `with-open`s in 3832c43.  Hope that's what you were looking for.